### ISSUE TITLE
Validators could have personal validation comment snippets

### ISSUE THREAD

**Issue by bgirardot** [hotosm/osm-tasking-manager2#886]
This would make it easy to offer very specific feedback and thanks, validators could easily and quickly point to the resources they think would best address the mapping they just validated.

**Comment by RAytoun** [comment]
I am happy to give thanks and positive feedback in the comments panel of the Tasking Manager. If I have anything that needs detailed or lengthy explanation then I send a message to that specific person via there OSM profile page so that no one else sees it and they will then get an email notifying them that they have a message.

**Comment by RAytoun** [comment]
Unfortunately it is a fact of life that mapping is not everyone's preference so there is a high volume of 'one off' mappers who do not return for ages. I am more inclined to take time to message newbies who have a couple of tiles completed rather than just a few edits. It shows that they are coming back to do more.

**Comment by bgirardot** [comment]
I like this a lot @RAytoun it suggests that completed task squares are a measure of commitment in your decision process for valuable and limited validation and feedback time.

my only comment is that task square "quality" varies so much many people who would have done 3 or 4 small, well done task squares will never complete some of these huge, map everything task squares so we need to take that into account. changesets might be more useful measure for commitment than task squares.

**Comment by RAytoun** [comment]
I agree with the number of days mapping as a better guide as this gives the activity of that specific mapper without interference. By interference I mean that you may not always be able to tell by other means who was the author of a mapped feature as this can be disguised by the last mapper who modified or amended it.
